index.js

```diff
@@ -137,6 +137,8 @@ function SendStream(req, path, options) {
   if (!this._root && opts.from) {
     this.from(opts.from)
   }
+
+  mime.default_type = opts.defaultType || null;
```

Comment:
The `mime` variable is a static variable, shared between all instances of `send`. Setting this off an option does not make sense, because executing two `send` commands with two different `defaultType` options will result in both requests having the `defaultType` that was send in the second call to `send`.